editoast/src/models/paced_train.rs

```diff
@@ -101,3 +101,25 @@ impl From<PacedTrain> for PacedTrainBase {
         }
     }
 }
+
+impl From<PacedTrain> for TrainSchedule {
```

Comment:
This will conflict with https://github.com/OpenRailAssociation/osrd/pull/11022 since it has changed after this nice [comment](https://github.com/OpenRailAssociation/osrd/pull/11022#discussion_r1983459662).